app/api/helpers/special_fields.py

```diff
@@ -27,17 +27,9 @@ class EventSubTopicField(fields.ChoiceString):
 
     def __init__(self, **kwargs):
         super(EventSubTopicField, self).__init__(
-            choice_list=[],
+            choice_list=DataGetter.get_event_subtopics().get(topic, []),
             **kwargs)
 
-    def validate(self, value):
```

Comment:
@Saptaks Sorry, I was testing some other things and accidentally pushed them. Same for the print statement